articles/native-platforms/ios-objc/index.yml

```diff
@@ -17,3 +17,11 @@ alias:
 seo_alias: ios-objc
 articles:
   - 01-login
+  - 02-custom-login
+  - 03-session-handling
+  - 04-user-profile
+  - 05-linking-accounts
+  - 06-rules
+  - 07-authorization 
+  - 08-calling-apis
+  - 09-mfa
```

Comment:
@sebacancinos The file is named `09-MFA.md` and it's listed in the index as `09-mfa.md`. This throws an exception upon deployment. For consistency, since the doc names are not capitalised, I recommend changing the filename, and not the index entry.